tests/integration/sync_test.go

```diff
@@ -52,7 +52,7 @@ func testKongState(
 		t.Errorf(err.Error())
 	}
 	opt := []cmp.Option{
-		cmpopts.IgnoreFields(kong.Service{}, "ID", "CreatedAt", "UpdatedAt"),
+		cmpopts.IgnoreFields(kong.Service{}, "ID", "CreatedAt", "UpdatedAt", "Enabled"),
```

Comment:
code smell: we will have new properties being defined all the time. Our tests should handle these.

Reply:
yeah, I will take care of this in the other "test" PR along with other test-related changes.